components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/OAuthScopeCache.java

```diff
@@ -72,7 +72,7 @@ public Scope getValueFromCache(OAuthScopeCacheKey key) {
         Scope entry = null;
         if (IdentityUtil.getIdentityCacheConfig(IDENTITY_CACHE_MANAGER, OAUTH_SCOPE_CACHE_NAME).isEnabled()) {
             entry = super.getValueFromCache(key);
-            if (log.isDebugEnabled()) {
+            if (log.isDebugEnabled() && entry != null) {
                 log.debug("Scope is getting from the cache. \n" + entry.toString());
```

Comment:
Use String.valueOf() instaid of null check.

Reply:
done